Error caused by NCC dependancy?

We are on KICS action version 1.6. It seems like an underlying dependancy of KICS has been updated and broken the KICS action. I believe this dependancy is NCC and the related issue from their repo is here https://github.com/vercel/ncc/issues/805

```
> kics-github-action@1.0.0 build
> ncc build src/main.js

ncc: Version 0.31.1
ncc: Compiling file index.js into CJS
Error: error:0308010C:digital envelope routines::unsupported
    at new Hash (node:internal/crypto/hash:71:19)
    at Object.createHash (node:crypto:133:10)
    at hashOf (/app/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:37:1855992)
    at ncc (/app/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:37:1860457)
    at runCmd (/app/node_modules/@vercel/ncc/dist/ncc/cli.js.cache.js:1:55128)
    at 819 (/app/node_modules/@vercel/ncc/dist/ncc/cli.js.cache.js:1:51698)
    at __webpack_require__ (/app/node_modules/@vercel/ncc/dist/ncc/cli.js.cache.js:1:5[904](https://github.com/HylandSoftware/Hyland.Experience.Iam.Api/actions/runs/4419152099/jobs/7747213148#step:6:905)8)
    at /app/node_modules/@vercel/ncc/dist/ncc/cli.js.cache.js:1:59260
    at /app/node_modules/@vercel/ncc/dist/ncc/cli.js.cache.js:1:59321
    at Object.<anonymous> (/app/node_modules/@vercel/ncc/dist/ncc/cli.js:8:28) {
  opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
}
node:internal/modules/cjs/loader:1078
  throw err;
  ^

Error: Cannot find module '/app/dist/index.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)
    at Module._load (node:internal/modules/cjs/loader:[920](https://github.com/HylandSoftware/Hyland.Experience.Iam.Api/actions/runs/4419152099/jobs/7747213148#step:6:921):27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```

--- a/package.json
+++ b/package.json
@@ -27,7 +27,7 @@
     "moment": "^2.29.4"
   },
   "devDependencies": {
-    "@vercel/ncc": "^0.31.1",
+    "@vercel/ncc": "^0.36.1",
     "prettier": "^2.4.1"
   }
 }

